compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -333,7 +333,8 @@ impl fmt::Display for BoundConstness {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
             Self::NotConst => f.write_str("normal"),
```

Comment:
why are we calling this "normal"

Reply:
where is this `Display` impl even used? if it's not used, pls remove instead

Reply:
It is used, in the error reporting for `TypeError::ConstnessMismatch`.

Reply:
oh ok